### PR TITLE
Password Strength

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[sc-9447]

File: packages/quill/src/app/components/FormPasswordStrength.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  Value extends T[Key] | T[Key][keyof T[Key]],
+  State extends DefaultState = DefaultState,
+  Error extends string = DefaultError
+>({
+  form,
+  name,
+  ...rest
+}: Omit<Omit<BoxProps, 'form'> & FormInputProps<T, State, Error, Key, Value>, 'as'>) => {
+  const { error, value } = useListener(form, name);
+  const result = zxcvbn((value as unknown) as string);
```
Comment (Mrtenz) [inline packages/quill/src/app/components/FormPasswordStrength.tsx:19]: Maybe memoize this

File: packages/quill/src/app/screens/CreatePassword.tsx [hunk 2]
```diff
@@ -68,28 +70,9 @@ export const CreatePassword = () => {
...
-              <ListItem mb="0" color="inherit">
-                {translateRaw('PASSWORD_CRITERIA_4')}
-              </ListItem>
-              <ListItem mb="0" color="inherit">
-                {translateRaw('PASSWORD_CRITERIA_5')}
-              </ListItem>
-            </List>
+            <FormPasswordStrength form={form} name="password" />
           </Box>
+          <Box width="100%" mt="2" color="BLUE_GREY"></Box>
```
Comment (Mrtenz) [inline packages/quill/src/app/screens/CreatePassword.tsx:75]: What's this for?

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: packages/quill/src/app/screens/CreatePassword.tsx (hunk 2 above)
Comment (FrederikBolding) [reply packages/quill/src/app/screens/CreatePassword.tsx:75]: Remnant of old code - removing

File: packages/quill/src/app/components/FormPasswordStrength.tsx (hunk 1 above)
Comment (FrederikBolding) [reply packages/quill/src/app/components/FormPasswordStrength.tsx:19]: Good idea

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**